python/plotting.py

```diff
@@ -337,6 +337,8 @@ def plot_data_mc(self, inclusive, channel, region, fig, var, gs, year='2016', no
         plt1.set_xlabel('')
         if 'dnn_score' in var:
             plt1.set_xlim(0, parameters["dnn_max"][year])
+        if 'bdt_score' in var:
```

Comment:
This file is an obsolete version of the plotter so I will remove it at some point